### ISSUE TITLE
SCTD_STRIKEOUT Tab Draw Mode Unsupported

### ISSUE THREAD

**Issue by orbitalquark** [orbitalquark/scinterm#3]
When SCI_SETTABDRAWMODE(SCTD_STRIKEOUT) is set, tabs are still drawn as arrows
(SCTD_LONGARROW).

**Comment by orbitalquark** [comment]
Fixed via 6a774158d8a3c7bc7ea120bc01cdb016fa351a7e

**Comment by orbitalquark** [comment]
The upstream patch referenced in the commit is as follows:
```
diff -r 22b6bbb36280 src/EditView.cxx
--- a/src/EditView.cxx	Sat Sep 05 07:55:08 2020 +1000
+++ b/src/EditView.cxx	Fri Oct 02 19:34:46 2020 -0400
@@ -1951,7 +1951,7 @@
 							if (!customDrawTabArrow)
 								DrawTabArrow(surface, rcTab, segmentTop, vsDraw);
 							else
-								customDrawTabArrow(surface, rcTab, segmentTop);
+								customDrawTabArrow(surface, rcTab, segmentTop, vsDraw);
 						}
 					}
 				} else {
diff -r 22b6bbb36280 src/EditView.h
--- a/src/EditView.h	Sat Sep 05 07:55:08 2020 +1000
+++ b/src/EditView.h	Fri Oct 02 19:34:46 2020 -0400
@@ -40,7 +40,7 @@
 void DrawStyledText(Surface *surface, const ViewStyle &vs, int styleOffset, PRectangle rcText,
 	const StyledText &st, size_t start, size_t length, DrawPhase phase);
 
-typedef void (*DrawTabArrowFn)(Surface *surface, PRectangle rcTab, int ymid);
+typedef void (*DrawTabArrowFn)(Surface *surface, PRectangle rcTab, int ymid, const ViewStyle &vsDraw);
 
 class LineTabstops;
 
```
